content/references/http-websocket-apis/public-api-methods/server-info-methods/nft_info.md

```diff
@@ -0,0 +1,132 @@
+      "This server may be out of date"
+    ]
+  }
+```
+
+*JSON-RPC*
+
+```json
+{
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
+  }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format][], with a successful result containing an `nft_info` response object with some arrangement of the following fields:
+
+| `Field`                             | Type            | Description          |
+|:------------------------------------|:----------------|:---------------------|
+| `nft_id`                            | String          | A unique identifier for the non-fungible token (NFT). |
+| `ledger_index`                      | Integer         | The [ledger index][] of the most recent ledger version where the state of this NFT was modified, as in the NFT was minted (created), changed ownership (traded), or burned (destroyed). The information returned contains whatever happened most recently compared to the requested ledger. |
+| `owner`                             | String          | The account ID of this NFT's owner at this ledger index. |
+| `is_burned`                         | Boolean         | Returns `true` if the NFT is burned at this ledger, or `false` otherwise. |
+| `flags `                            | Integer         | The flag set of this NFT. |
+| `transfer_fee`                      | Integer         | The transfer fee of this NFT. See [NFTokenMint Fields](nftokenmint.html#nftokenmint-fields) for more information on transfer fees. |
+| `issuer`                            | String          | The account ID which denotes the issuer of this NFT. |
+| `nft_taxon`                         | Integer         | The NFT’s taxon. |
+| `nft_sequence`                     | Integer         | The NFT’s sequence number. |
+| `uri`                               | String          | _(Omitted if the NFT is burned at this ledger.)_. This field is NULL if the NFT is not burned at this ledger but does not have a URI.  If the NFT is not burned at this ledger and it does have a URI, this field is a string containing the decoded URI of the NFT. NOTE: If you need to retrieve the URI of a burnt token, re-request `nft_info` for this token, specifying the `ledger_index` as the one previous to the index where this token was burned ({ledger_index-where-token-was-burned} - 1). |
```

Comment:
"NULL" does not need to be in all caps, but the **Type** column should probably read `String or null` if this can actually return null.